### PR TITLE
Create structure of audiomack

### PR DESCRIPTION
**Pull request by Cxristian**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -2,6 +2,6 @@
 browser=chrome
 
 #Properties for GUI
-login=https://audiomack.com/
-username=
-password=
+login=https://exampletest-trials653.orangehrmlive.com/
```
Comment (alszla) [inline gradle.properties:5]: You're not supposed to upload authenticatio credentials.

File: src/main/java/core/selenium/webDrivers/Edge.java [hunk 2]
```diff
@@ -20,4 +20,4 @@
 //        WebDriverManager.getInstance(EDGE).setup();
```
Comment (alszla) [inline src/main/java/core/selenium/webDrivers/Edge.java]: No need to keep commented blocks.

File: src/main/java/orangeHRM/PageTransporter.java [hunk 3]
```diff
@@ -10,12 +10,13 @@
  * with Jala Foundation.
  */
 
-package audiomack;
+package orangeHRM;
 
 import core.selenium.WebDriverManager;
 import core.utils.ReaderPropertiesFile;
+import orangeHRM.ui.pages.DashboardPage;
```
Comment (alszla) [inline src/main/java/orangeHRM/PageTransporter.java:17]: Import statements are mixed. Please group them accoirding their family.

File: src/main/java/orangeHRM/entities/Courses.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    public String getTitle() {
+        return title;
+    }
+
+    /**
+     * Sets the title of Course.
+     *
+     * @param title for the account.
+     */
+    public void setTitle(String title) {
```
Comment (alszla) [inline src/main/java/orangeHRM/entities/Courses.java]: Shouldn't method parameters be final?

File: gradle.properties (hunk 1 above)
Comment (Cxristian) [reply gradle.properties:5]: ok

File: src/main/java/core/selenium/webDrivers/Edge.java (hunk 2 above)
Comment (Cxristian) [reply src/main/java/core/selenium/webDrivers/Edge.java]: ok i deleted it

File: src/main/java/orangeHRM/entities/Courses.java (hunk 4 above)
Comment (Cxristian) [reply src/main/java/orangeHRM/entities/Courses.java]: ok

**Review by alszla — APPROVED** [review]

**Pull request merged**